packages/react/src/components/DatePicker/next/DatePicker-story.js

```diff
@@ -0,0 +1,167 @@
+/**
```

Comment:
This file should be `DatePicker-stories.js`, right? I didn't think we wanted duplicate story files in `next` folders

Reply:
You're probably right. Tbh I've been really confused with next/current stories. So should I just delete this file? @tay1orjones

Reply:
@jnm2377 Yeah no sweat! `*-stories.js` are what's pulled into the new storybook and should use the `next` version of components.

So I think just make sure every example in this `next/DatePicker-story.js` is already represented/included in the `next/DatePicker-stories.js` and then you can delete this `next/DatePicker-story.js`.

I'd also double check the imports in `next/DatePicker-stories.js` to be sure it's importing the `next` version of DatePicker, DatePickerInput, etc.